### PR TITLE
Decrease default texture quality

### PR DESCRIPTION
**Pull request by konraddysput**
# Why

This diff decreases the default screenshot quality generated by backtrace-unity when a user checks options to include screenshots in each backtrace-report.

### CODE REVIEW

**Review by rqbacktrace — APPROVED** [review]
Looks good to me! We should make screenshot quality minimal by default, and customer can increase it if they want but they should be warned it might cause issues on the server.

**Comment by konraddysput** [comment]
We need to add more changes to always apply screenshot quality settings. Please ignore this pull request for now.

File: Runtime/Model/Database/BacktraceDatabaseAttachmentManager.cs [hunk 1]
```diff
@@ -99,55 +99,50 @@ private string GetScreenshotPath(string dataPrefix)
...
 
 #if UNITY_2019_1_OR_NEWER
-                        RenderTexture screenTexture = RenderTexture.GetTemporary(Screen.width, Screen.height);
-                        ScreenCapture.CaptureScreenshotIntoRenderTexture(screenTexture);
+                    RenderTexture screenTexture = RenderTexture.GetTemporary(Screen.width, Screen.height);
+                    ScreenCapture.CaptureScreenshotIntoRenderTexture(screenTexture);
 #else
-                        Texture2D screenTexture = ScreenCapture.CaptureScreenshotAsTexture();
+                    
+Texture2D screenTexture = ScreenCapture.CaptureScreenshotAsTexture();
```
Comment (rqbacktrace) [inline Runtime/Model/Database/BacktraceDatabaseAttachmentManager.cs:113]: Formatting issue and extra newline

File: Runtime/Model/Database/BacktraceDatabaseAttachmentManager.cs [hunk 2]
```diff
@@ -99,55 +99,50 @@ private string GetScreenshotPath(string dataPrefix)
...
+                    RenderTexture.active = previousActiveRT;
 
-                        RenderTexture.ReleaseTemporary(rt);
+                    RenderTexture.ReleaseTemporary(rt);
 
 #if UNITY_2019_1_OR_NEWER
-                        RenderTexture.ReleaseTemporary(screenTexture);
+                    RenderTexture.ReleaseTemporary(screenTexture);
 #else
                         GameObject.Destroy(screenTexture);
```
Comment (rqbacktrace) [inline Runtime/Model/Database/BacktraceDatabaseAttachmentManager.cs:144]: You can decrease indent level here

**Review by rqbacktrace — APPROVED** [review]
Approved with minor formatting issues!

File: Runtime/Model/Database/BacktraceDatabaseAttachmentManager.cs (hunk 1 above)
Comment (konraddysput) [reply Runtime/Model/Database/BacktraceDatabaseAttachmentManager.cs]: fixed. thx

File: Runtime/Model/Database/BacktraceDatabaseAttachmentManager.cs (hunk 2 above)
Comment (konraddysput) [reply Runtime/Model/Database/BacktraceDatabaseAttachmentManager.cs]: fixed. thx!

**Pull request merged**